docs/telemetry.md

```diff
@@ -22,32 +94,165 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
+
+The Percona Telemetry agent functions as a dedicated OS daemon process. The service name on the operating system is `percona-telemetry-agent`. The agent can create, read, write, and delete files in the `${telemetry root path}` and only interacts with JSON files.
+
+You can find the Telemetry agent log file at `/var/log/percona/telemetry-agent.log`.
+
+The Telemetry agent is a mandatory dependency to Percona Server for MySQL packages. Deleting the agent also deletes the database. 
```

Comment:
Please also add re packages removal if TA is installed as a dependency (this is based on packaging system behavior not smth special that we did):
- On RPM systems the TA package should be automatically removed when the last dependency package is removed.
- On deb-based systems (Ubuntu and Debian) the Telemetry Agent will **NOT** be automatically deleted when the DB package is removed. To delete Telemetry Agent together with DB the `--autoremove` should be passed but this automatically removes all the dependencies. ( https://perconadev.atlassian.net/browse/PKG-34 ).

The same applies to Centos 7 (yum autoremove should be used) but I am not sure whether we will release on Centos7 or no.